repo/packages/S/storm/0/package.json

```diff
@@ -8,7 +8,7 @@
   "framework": true,
   "tags": ["streaming"],
   "postInstallNotes": "Storm is installed in your cluster.",
-  "preInstallNotes": "Note that the Apache Storm DCOS Service is beta and there may be bugs, incomplete features, incorrect documentation or other discrepancies.",
+  "preInstallNotes": "This DC/OS Service is currently in preview and there may be bugs, incomplete features, incorrect documentation or other discrepancies.",
```

Comment:
If we keep this sentence and we add this check to our CI, we will be looking for `preInstallNotes` that start with "This DC/OS Service is currently in preview". Notice no period.